base/Makefile

```diff
@@ -66,6 +66,7 @@ endif
 	@echo "const DOCDIR = \"$(docdir_rel)\"" >> $@
 	@echo "const LIBDIR = \"$(libdir_rel)\"" >> $@
 	@echo "const INCLUDEDIR = \"$(includedir_rel)\"" >> $@
+	@echo "const VERSDIR = \"$(VERSDIR)\"" >> $@
```

Comment:
I have defined `Base.VERSDIR` since it appears in quite a few places now.

Should you later decide to switch from `v$(MAJOR).$(MINOR)` to `v$(MAJOR)`, this would not require further changes.